.travis.yml

```diff
@@ -3,11 +3,11 @@ python:
   - "2.7"
   - "3.6"
 install:
-  - pip install flake8
+  - pip install flake8 pytest
   - pip install -r requirements.txt
 before_script:
   - flake8 . --count --max-complexity=11 --max-line-length=85 --show-source --statistics
 script:
-  - echo success
+  - pytest --capture=sys
```

Comment:
What about `python -m pytest`

Reply:
In a Travis virtualenv, __pip__ is set up to point to the Python version that we are testing.

This goes back to what is "pip" on line 6.
* In Python 2.7 it is [pip 9.0.1 from /home/travis/virtualenv/python2.7.13/lib/python2.7/site-packages (python 2.7)](https://travis-ci.org/fossasia/query-server/jobs/287833510#L576)
* In Python 3.6 it is [pip 9.0.1 from /home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages (python 3.6)](https://travis-ci.org/fossasia/query-server/jobs/287833511#L553)

While we could make the change that you suggest, it does not really help or hurt.  The same could be done to the flake8 commands but again it does not really help or hurt (except it adds a bit of complexity for the reader).

Thoughts?

Reply:
OK, Please check on Codacy. It should pass.

Reply:
> How do I "check on Codacy"?

Please make sure that Codacy passes. Thanks